Sources/Sentry/SentrySDK.m

```diff
@@ -632,6 +632,25 @@ + (void)stopProfiler
 
     [SentryContinuousProfiler stop];
 }
+
++ (void)startProfileSession
```

Comment:
`m`: What happens if the user didn't start the SDK yet? Should we check that and make this a no op? The same applies for stop.

Reply:
It's a no-op because we try to check `[currentHub.client.options isContinuousProfilingEnabled]` but `currentHub.client.` is `nil` if the SDK is closed. I added a test for that in https://github.com/getsentry/sentry-cocoa/pull/4961